addons/hr/models/hr_employee.py

```diff
@@ -124,11 +123,23 @@ class HrEmployeePrivate(models.Model):
         ('user_uniq', 'unique (user_id, company_id)', "A user cannot be linked to multiple employees in the same company.")
     ]
 
-    def _get_placeholder_filename(self, field=None):
-        image_fields = ['image_%s' % size for size in [1920, 1024, 512, 256, 128]]
-        if field in image_fields:
-            return 'hr/static/src/img/default_image.png'
-        return super()._get_placeholder_filename(field=field)
+    def _compute_avatar_1920(self):
+        return self._compute_avatar_xxxx('1920')
+
+    def _compute_avatar_128(self):
+        return self._compute_avatar_xxxx('128')
+
+    def _compute_avatar_xxxx(self, field):
+        for record in self:
+            if record['image_%s' % field]:
+                record['avatar_%s' % field] = record['image_%s' % field]
+            else:
+                record['avatar_%s' % field] = record.generate_avatar_svg(record.name) if not record.user_id.share else record._get_placeholder()
+
+    def _get_placeholder(self):
+        image = 'company_image.png' if self.is_company else 'avatar_grey.png'
+        image_path = get_resource_path('base', 'static/img', image)
+        return base64.b64encode(file_open(image_path, 'rb').read())
```

Comment:
same remark as in partner, also there is no check for is_company needed on employee